packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -434,6 +436,9 @@ export class GarbageCollector implements IGarbageCollector {
 	// This is the version of GC data in the latest summary being tracked.
 	private latestSummaryGCVersion: GCVersion;
 
+	// Feature Support info persisted to this container's summary
+	private readonly persistedGcFeatureMatrix: GCFeatureMatrix | undefined;
```

Comment:
nit: persistedGCFeatureMatrix

Reply:
Since Navin also agrees that `Gc` is better (I like it better) I'm leaving it.  We can take a pass to clean up if we care... I don't....